src/dbus-proxy/CMakeLists.txt

```diff
@@ -7,7 +7,8 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Core REQUIRED)
+# NOTE: This project will be deprecate in the feature, but maybe will still needed, now it cannot be compiled with qt6
```

Comment:
```suggestion
# NOTE: While this project is deprecated, it might still be needed for compatibility reasons. Currently it cannot be compiled with Qt6
```